### DISCUSSION TITLE
Why doesn't Web Search offer picking more browsers?

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#118744] [Mobile] [answered]
Here we are in the mobile app. We highlight some text. Then as you see, we choose Web Search. 

![Screenshot_20240412_070208_GitHub.jpg](https://github.com/community/community/assets/1230959/23384c06-b134-47a1-bb94-aabfd2ad1e1e)

Then what happens is odd. We are only offered a choice of two of the apps installed on her phone. Where is our favorite browser? Or any other things that could respond to this Android _intent_.

![Screenshot_20240412_070309_Android System.jpg](https://github.com/community/community/assets/1230959/184cae94-b88a-41b2-9fe4-e59abdfb5b38)

I conclude that somehow this function has been hijacked by the GitHub app and it's not offering all the possibilities that it should. 

Yes we set our default browser in the Android settings. We are using Android 13 on a Samsung Galaxy.

**Comment by stevepopovich** [comment]
Hi @jidanni not sure I understand the expected behavior here. I can confirm that we are not intercepting this intent at all, and this is just what the OS shows. In my testing, the bottom sheet shows all the browsers installed on the device. I believe only browsers can intercept these intents, so it is expected that only installed browsers show up

**Reply by jidanni** [reply]
Even the mighty Chrome Beta that I use all day doesn't show up. Well I guess it will remain a mystery.

**Reply by stevepopovich** [reply, marked as answer]
Ok, interesting to know. I will file this internally just to note it, let me know if anything changes here. Thank you.
